ebml/EbmlElement.h

```diff
@@ -234,32 +234,31 @@ class DllApi x : public BaseClass { \
 
 #define EBML_CONCRETE_CLASS(Type) \
     public: \
-        libebml::EbmlElement & CreateElement() const override {return Create();} \
```

Comment:
This doesn't work for me. There are places where I have a generic `EbmlElement`, and I need to create another instance of the same type. Before I could just use this function; now I cannot see a way to do it anymore.

Reply:
Indeed, we need this for generic usage. I dropped this commit.